src/services/baseService.ts

```diff
@@ -81,4 +87,33 @@ export abstract class BaseService {
   protected slsConfigFile(): string {
     return ("config" in this.options) ? this.options["config"] : "serverless.yml";
   }
+
+  private setDefaultValues(): void {
+    // TODO: Right now the serverless core will always default to AWS default region if the
+    // region has not been set in the serverless.yml or CLI options
+    const awsDefault = "us-east-1"
```

Comment:
nit: feels odd to have the const within the function vs. more global -- both for the `awsDefault` and probably makes sense to have something like `azureDefault` that sits alongside it, so it's more clear what it's being used for.  Comment definitely helps.

Reply:
this won't be necessary anymore once serverless merge the other PR that remove the AWS default

Reply:
We are only referencing these in this one function right now.  Agree if we needed to get the default values in other places we can bump this up.  Going to keep as is for the time being.